site/zenodo_rdm/migrator/transform.py

```diff
@@ -237,8 +237,8 @@ def _profile(self, entry):
     def _preferences(self, entry):
         """Returns the preferences."""
         return {
-            "visibility": "private",
-            "email_visibility": "private",
+            "visibility": "restricted",
```

Comment:
Small comment: it was recently discussed whether the default's profile visibility should be configured per instance (e.g. a config). 

https://github.com/inveniosoftware/invenio-app-rdm/issues/1779

Not blocking, just leaving a notice for later.

Reply:
Transform will be hardcoded per instance, and since we're bypassing the service/schemas config wouldn't affect this class.

The plan is in general to leave the transform/extract logic free from any application logic, so that it can be fast/simple and focus on data mangling.

Validation (of different levels/types) will anyways take place on the Load step.